### PR TITLE
Repack

### PR DESCRIPTION
**Pull request by HadesArchitect**
General update, plugin compliance, better readability, build automation.

**What was done**
- Restructured source code
- Removed SimpleJson
- Added Grunt
- Improved Readme
- Added license
- Added plugin required files
- Added logo

**How to test**
- Review the changeset
- Clone locally 
- Remove old node_modules and docker containers
- Build & launch the project following the README.md instructions
- Check for warnings in logs (should be no warnings)
- In grafana logs should be `msg="Registering plugin" logger=plugins name="Apache Cassandra"`
- Check that grafana is running and the plugin is available in a local DS list

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,9 @@
+MIT License
+
+Copyright (c) 2020 Aleksandr Volochnev, Aleksey Slaviansky
```
Comment (DevAleks) [inline LICENSE]: Error, need to change Aleksei Slavianskii

Comment (HadesArchitect) [reply LICENSE]: Fixed

**Comment by DevAleks** [comment]
Review ok)

**Comment by DevAleks** [comment]
Test ok!

**Pull request merged**